docs/explorers.md

```diff
@@ -1 +1,25 @@
 # Explorers
+##Official Explorers  
+
+###Cardano Foundation  
+####Mainnet  
+https://explorer.cardano.org/  
+
+####ITN  
+https://itnexplorer.cardano.org/  
+
+###Emurgo  
+####Seiza (Mainnet)  
+https://seiza.com/  
+
+###IOHK   
+####ITN  
```

Comment:
Can we remove ITN? ITN rewards run out on epoch # 189, and the explorers are hardly being maintained - like this one is already been down for a few days, we can possibly add testnet explorers later, but for now lets try and keep support faq limited to mainnet.
Testnet related queries are usually gonna end up in Dev group (if development related) or stakepool workgroup